setup.py

```diff
@@ -1,44 +1,28 @@
-import distutils
 import os
 import shutil
 import subprocess
 import sys
 
 from setuptools import find_packages, setup
+from setuptools.command.build_ext import build_ext as build_ext_orig
 
 import versioneer
 
 
-class MavenCommand(distutils.cmd.Command):
-    """Run the maven build command"""
-
-    description = "run the mvn install command"
-    user_options = []
-
-    def initialize_options(self):
-        """No options"""
-        pass
-
-    def finalize_options(self):
-        """No options"""
-        pass
+class build_ext(build_ext_orig):
+    """Build the external Java libraries"""
 
     def run(self):
         """Run the mvn installation command"""
-        # We need to explicitely specify the full path to mvn
-        # for Windows
         maven_command = shutil.which("mvn")
         if not maven_command:
             raise OSError(
                 "Can not find the mvn (maven) binary. Make sure to install maven before building the jar."
             )
         command = [maven_command, "clean", "install", "-f", "pom.xml"]
-        self.announce(f"Running command: {' '.join(command)}", level=distutils.log.INFO)
 
         subprocess.check_call(command, cwd="planner")
 
-        # Copy the artifact. We could also make maven do that,
-        # but in this way we have full control from python
         os.makedirs("dask_sql/jar", exist_ok=True)
         shutil.copy("planner/target/DaskSQL.jar", "dask_sql/jar/DaskSQL.jar")
```

Comment:
Think we still need to call the base class's `run`

```suggestion
        shutil.copy("planner/target/DaskSQL.jar", "dask_sql/jar/DaskSQL.jar")

        super().run()
```